packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/NotOwnedSandboxInfo/Summary.tsx

```diff
@@ -47,13 +51,16 @@ export const Summary = () => {
     forkedTemplateSandbox,
     tags,
     team,
+    privacy,
   } = currentSandbox;
   const {
     editor: { forkSandboxClicked },
   } = useActions();
+  const { isPro } = useWorkspaceSubscription();
 
   const isForked = forkedFromSandbox || forkedTemplateSandbox;
   const { url: templateUrl } = getTemplateDefinition(template);
+  const isUnlistedOrPrivate = privacy === 1 || privacy === 2;
```

Comment:
Unrelated thought: at some point, I think it would be good to create an enum to map the privacy options.